core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -1,73 +1,80 @@
 {% extends "base.html" %}
 
 {% block maintitle %}
-  Oppia Editor
+Oppia Editor
```

Comment:
The existing indentation was correct -- please don't change it. We optimize for readability of this file over the HTML shown in the browser console.